public/css/styles.css

```
@@ -248,3 +250,84 @@ html, body {
     text-align: center;
     padding: 15px;
 }
+
+/* Alerts */
+.alert-error {
+    display: block;
+    margin-top: -7px;
+    margin-bottom: 17px;
+    text-transform: none;
+    font-size: 12px;
+    font-weight: normal;
+    color:rgb(219, 0, 48);
```

```suggestion
    color: rgb(219, 0, 48);
```
Não dá para colocar um hexadecimal? Acho que ficar melhor padronizado.